packages/vanilla/src/components/footer/README.md

```diff
@@ -0,0 +1,119 @@
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+import '@carbon/ibmdotcom-styles/scss/components/footer/index.scss';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Footer } from '@carbon/ibmdotcom-react';
```

Comment:
@andysherman2121 this is referencing React. Also, I think that the initialize example below should suffice for the description for the non-data fetching. You should probably bring up the data fetch approach up, and even I would say that should be the preferred method over the pure template one.

Reply:
@jeffchew is there any reason they should be using the pure template approach ever?

Reply:
some teams might want to use it. Drupal team for instance is doing that for the Masthead already, and is asking to do that for the footer too. They might want to have their own way of fetching the navigation data and injecting it.

Reply:
got it